BlueprintUI/Tests/BuilderTests.swift

```diff
@@ -174,7 +174,7 @@ class BuilderTests: XCTestCase {
                 "2"
             }
 
-            if #available(iOS 11, *) {
+            if #available(iOS 14, *) {
```

Comment:
Why is it still useful to keep any of these `if #available(iOS 14, *)` checks if you cannot deploy on a version below iOS 14?

Reply:
This is part of ensuring our function builder syntax for availability checks is working properly.

Reply:
```
  /// If declared, this will be called on the partial result of an 'if
  /// #available' block to allow the result builder to erase type
  /// information.
  static func buildLimitedAvailability(_ component: Component) -> Component { ... }
```

Reply:
ohhh, got it.